crates/frontend/src/pages/context_override.rs

```diff
@@ -326,6 +342,22 @@ pub fn context_override() -> impl IntoView {
                     }}
 
                 </div>
+
+                <Show when=move || modal_visible.get()>
```

Comment:
@sauraww Can you make this a separate module for future usage?

Reply:
And make the tailwind a bit cleaner to read by separating out common ones, hard to read those.

Reply:
Agreed on making this a component that can be quickly used since we have deletes in multiple screens